### ISSUE TITLE
Release Encryption 1.5.2

### ISSUE THREAD

**Issue by IljaN** [owncloud/encryption#338]
- Add increment range option (https://github.com/owncloud/encryption/pull/274)
- Library bumps

**Comment by IljaN** [comment]
This does not touch any risky parts of encryption, an additional option is added to a cli command.

**Comment by pako81** [comment]
to note that encryption is currently broken on distro coming with openSSL v3.x and needs to be manually workarounded: https://github.com/owncloud/core/issues/40071#issuecomment-1133422548

**Comment by jnweiger** [comment]
Shoud we do a small release with only this fix?
Or is there something quick, we could also add concerning the ubuntu 22.04 problem? 
At least a prominent message should be printed, that a required cypher is not available + some instructions as compiled in core#40071

**Comment by IljaN** [comment]
@jnweiger imo openssl is out of scope for this one. Would probably need to be a 1.6.0 or maybe even 2.0.0 (still analyzing)

**Comment by jnweiger** [comment]
@GeraldLeikam RC1 is built:
https://github.com/owncloud/encryption/releases/download/v1.5.2-rc1/encryption-1.5.2-rc1.tar.gz

**Comment by jnweiger** [comment]
## Issues seen while testing
* [ ] https://github.com/owncloud/encryption/issues/342 (not a regression, not a blocker)

**Comment by GeraldLeikam** [comment]
Release in the marketplace does not work because the info.xml has no <summary> field.

![image](https://user-images.githubusercontent.com/54261654/172763903-0954f46f-69cb-428c-b565-eaf2615b936c.png)

**Comment by IljaN** [comment]
We probably should ask PM to come up with text.

**Comment by GeraldLeikam** [comment]
Seems like encryption is an app that is not released in the marketplace because it is part of core. sorry it was my mistake

**Comment by phil-davis** [comment]
Yes, encryption is bundled in the regular "minimal" release tarball, and some other apps like configreport, firstrunwizard, market, notifications,...

And the "complete" release tarball has even more apps bundled. But that is a different beast.
